frontend/app/src/App.tsx

```diff
@@ -132,6 +132,7 @@ import { AppNavigation, MaybeStateUpdate } from "./util/AppNavigation"
 export interface Props {
   screenCast: ScreenCastHOC
   theme: ThemeManager
+  stExecTimestamp: number
```

Comment:
nit: maybe it's a good idea to name this `streamlitExecutionStartedAt` to match the other name where it is passed to [here](https://github.com/streamlit/streamlit/pull/9702/files#diff-6538c9c4cabf23f29921a35872abf8dd4b8516b9bdba08a44138d09d4f9c1980R96). Also, `streamlitExecutionStartedAt` is a little bit more descriptive